### PR TITLE
Add docs of chrome API.

### PR DESCRIPTION
**Pull request by Hyperkid123**
jira: https://issues.redhat.com/browse/RHCLOUD-15039

screenshot-localhost_3000-2021 06 29-14_33_27

### CODE REVIEW

File: packages/docs/pages/chrome/chrome-api.md [hunk 1]
```diff
@@ -1 +1,324 @@
...
+// URL: https://ci.console.redhat.com/beta/insights/advisor/recommendations
+getApp() // => 'advisor'
+```
+
+### `getBundle`
+
+A function that will return the name of the current bundle:
+
+```jsx
+
```
Comment (rvsia) [inline packages/docs/pages/chrome/chrome-api.md:134]: empty line

**Review by rvsia — APPROVED** [review]

File: packages/docs/examples/chrome/api/app-action.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+const MyComponent = () => {
+    const { appAction } = useChrome();
+    const handleClick = () => {
+        appAction('create-entity');
+    };
+
+    return (
+        <button onClick={handleClick}>
+          Open create dialong
```
Comment (epwinchell) [inline packages/docs/examples/chrome/api/app-action.js:12]: dialog

File: packages/docs/pages/chrome/chrome-api.md [hunk 3]
```diff
@@ -1 +1,324 @@
...
+A function that redirects the browser to Red Hat SSO.
+
+#### `auth.logout`
+
+A function that logs out current user and redirects the browser to landing page.
+
+### `createCase`
+
+A function trough which a support case is create. By default, any support case contains following:
```
Comment (epwinchell) [inline packages/docs/pages/chrome/chrome-api.md:78]: through

File: packages/docs/pages/chrome/chrome-api.md [hunk 4]
```diff
@@ -1 +1,324 @@
...
+```js
+{
+  createdBy, // current user
+  environment, // current environtment (Production, Production beta)
+  product, // current product
+}
+```
+
+<br />
+In addition to the default data you can also add additional options:
```
Comment (epwinchell) [inline packages/docs/pages/chrome/chrome-api.md:89]: In addition to the default data,

**Comment by Hyperkid123** [comment]
@epwinchell thanks for the grammar check :100:.

**Pull request merged**